src/App.tsx

```diff
@@ -1,25 +1,40 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import Counter from "./components/Counter";
+import Search from "./components/SearchForm";
+import Genre from "./components/GenreSelect";
+import { useState } from "react";
 
 function App() {
+  const [selectedGenre1, setSelectedGenre1] = useState("Comedy");
+  const [selectedGenre2, setSelectedGenre2] = useState("Western");
+
+  function onSearch(searchText: string) {
+    console.log(searchText);
+  }
+
+  function onSelect1(genre: string) {
+    console.log(genre);
+    setSelectedGenre1(genre);
+  }
+
+  function onSelect2(genre: string) {
+    console.log(genre);
+    setSelectedGenre2(genre);
+  }
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <h1>Counters</h1>
+      <Counter initialValue={0}></Counter>
+      <Counter initialValue={10}></Counter>
+      <h1>SearchForm</h1>
+      <Search initialSearchText="Search" onSearch={onSearch}></Search>
+      <h1>GenreSelect</h1>
+      <Genre
+        genres={["ALL", "DOCUMENTARY", "COMEDY", "HORROR", "CRIME"]}
```

Comment:
The initial select in the component doesnt work, as you are passing "Comedy" but genres array contains 'COMEDY'.
1) use Typescript functionality to indicate such kind of problems on development stage
2) its a good practice to 'normalise' strings before compering (toLowerCase usualy)